notebooks/optimal-stopping-point/README.md

```diff
@@ -0,0 +1,30 @@
+# Optimal Stopping Point Prediction
+
+The aim of this ML problem is to predict an optimal stopping point for CI tests based on their test duration (runtimes). We perform an initial data analysis as well as feature engineering on the testgrid data. Furthermore, we also calculated the optimal stopping point by identifying the distribution of the test_duration values for different CI tests and comparing the distributions of passing/failing tests.
+
+## Dataset
+
+ In order to achieve the optimal stopping point, we would be looking into the  [testgrid](https://testgrid.k8s.io/) data for all the passing and failed tests and find the distribution type for the test_duration metric. The test_duration metric tracks the time it took for a test to complete its execution. We can visualize the distribution of the test_duration metric across various testgrid dashboards and jobs. Based on the distribution type identified, we can find a point after which the test has a higher probability of failing.
+
+## Feature Engineering
+
+After fetching the data, we approximate the distributions of the test_duration variable and also check its Goodness of fit for different TestGrid tests across all TestGrid dashboards and grids. Based on the type of distribution identified, we can calculate the probability of the test failing.
+
+We fetch data for all the passing and failed tests and find the distribution type for the `test_duration` metric. The `test_duration` metric tracks the time it took for a test to complete its execution. We can visualize the distribution of the test_duration metric across various testgrid dashboards and jobs. Based on the distribution type identified, we can find top two distributions based on betterment of fit and probability density plots are used to understand data distribution for a continuous variable and likelihood (or probability) of obtaining a range of values that the continuous variable can assume. The area under the curve contains the probabilities for the test duration values.
```

Comment:
`The area under the curve contains the probabilities for the test duration values.` -> I feel like this could be somewhat unclear for some readers. Could we rephrase a bit? Maybe like "in a test_duration PDF, the area under the curve from 0 to a given value represents the probability that the test_duration is less than or equal to that value" (pls correct me if im wrong)?